blockstore/pom.xml

```diff
@@ -46,9 +53,10 @@
             <version>${vcloud-director.version}</version>
         </dependency>
         <dependency>
+            <!-- TODO remove when brooklyn-server is upgraded to 2.0.2 or later. -->
             <groupId>org.apache.jclouds.labs</groupId>
             <artifactId>azurecompute-arm</artifactId>
-            <version>2.0.1</version>
+            <version>2.0.2-SNAPSHOT</version>
```

Comment:
Use the new `2.0.1.1-20170526.0937` azurecompute release. It's based on `io.cloudsoft.jclouds.labs` groupId so you'll have to exclude this one.